### DISCUSSION TITLE
Can we use Litellm to connect with different models ?

### DISCUSSION THREAD

**Discussion by Greatz08** [swirlai/swirl-search#1488] [Q&A] [answered]
if yes then are their steps for connecting with it ?

**Comment by sid-swirl** [comment]
Hi, the enterprise version supports LiteLLM, but community does not.

**Comment by sid-swirl** [comment, marked as answer]
Here's the doc for it: https://docs.swirlaiconnect.com/AI-Connect.html 

Ping me if you're interested in getting access. Thanks!!
